server/mailet/mailets/src/test/java/org/apache/james/transport/util/SizeUtilsTest.java

```diff
@@ -0,0 +1,75 @@
+ /****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.transport.util;
+
+import org.junit.Test;
+
+
+import static org.apache.james.transport.util.SizeUtils.humanSize;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class SizeUtilsTest {
+
+    @Test
+    public void humanSizeShouldAddByteUnitForSmallSize() {
+        assertThat(humanSize(1000)).isEqualTo("1000 B");
+    }
+
+    @Test
+    public void humanSizeShouldScaleToKilobyteWhenSizeIsBetweenKilobyteAndMegabyte() {
+        assertThat(humanSize(1024)).isEqualTo("1 KiB");
```

Comment:
What do you prefer KiB (1024) or KB (1000 B) ?

Reply:
KiB but i'm probably the only one (:

Reply:
No we are two

Reply:
nope ok for KiB too :+1: